src/Utilities/DOM.jl

```diff
@@ -298,5 +300,81 @@ function escapehtml(text::AbstractString)
     end
 end
 
+
+"""
+Provides support for conversion from `Markdown.MD` to `DOM.Node` objects.
+"""
+module MarkdownConverter
+
+import ..DOM: @tags, Node, Tag
+
+@tags blockquote strong code pre p a h1 h2 h3 h4 h5 h6 hr
+@tags img em span div br ul ol li table th tr td
+
+
+import Base.Markdown:
+
+    MD, BlockQuote, Bold, Code, Header, HorizontalRule,
+    Image, Italic, LaTeX, LineBreak, Link, List, Paragraph, Table
+
+
+"""
+Convert a markdown object to a `DOM.Node` object.
+
+The `parent` argument is passed to allow for context-dependant conversions.
+"""
+mdconvert(md) = mdconvert(md, md)
+
+
+mdconvert(text::AbstractString, parent) = text
```

Comment:
It gets converted automatically, but should this be explicitly `= Node(text)`, so that `mdconvert` would always return a `Node`?